components/patient-contacts/src/main/java/org/phenotips/data/ContactInfo.java

```diff
@@ -225,7 +225,7 @@ private void setEmails(List<String> emails)
             {
                 List<String> collectedEmails = new ArrayList<String>();
                 for (String email : emails) {
-                    for (String parsedEmail : StringUtils.split(email, ",|;")) {
+                    for (String parsedEmail : StringUtils.split(email, ",|;| ")) {
```

Comment:
Afaik the second parameter of `split` which specifies the separators is treated as a list of characters, not a regular expression. You should just add the space, there's no need for another `|`.

Reply:
I think Marta is right (https://github.com/phenotips/phenotips/pull/2815#issuecomment-460821455), and I would remove `|` altogether as a highly unlikely separator

Reply:
`|` serves as an `OR` in this case

Reply:
But it looks like it does not, from the specification of this function

Reply:
will remove it then